curs_main.c

```diff
@@ -1802,7 +1802,9 @@ int mutt_index_menu (void)
       case OP_MAIN_MODIFY_LABELS_THEN_HIDE:
       {
 	if (Context->magic != MUTT_NOTMUCH) {
-	  mutt_message (_("No virtual folder, aborting."));
+	  // remove remaining arguments
+	  mutt_get_field ("", buf, sizeof(buf), MUTT_NM_TAG);
```

Comment:
`buf[0] = '\0';` is probably there to truncate the input buffer size to zero.

I tried it instead of the `mutt_get_field` call and it brought back the original issue.